pkg/services/live/pushws/push.go

```diff
@@ -159,6 +157,9 @@ func (s *Handler) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// This will keep the same fields alive while the connection is open
+	converter := convert.NewConverterWithHistory()
```

Comment:
If the converter is scoped to a single websocket connection... do we even need the mutex?